config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet

```diff
@@ -8,13 +8,13 @@
             alert: 'Prow images are stale',
             expr: |||
               time()-max(prow_version) > %d * 24 * 3600
-            ||| % $._config.prowImageStaleByDays,
-            'for': '5m',
+            ||| % $._config.prowImageStaleByDays.daysStale,
+            'for': $._config.prowImageStaleByDays.hoursToTrigger,
             labels: {
               severity: 'critical',
             },
             annotations: {
-              message: '@test-infra-oncall The prow images are older than %d days.' % $._config.prowImageStaleByDays,
+              message: 'The prow images are older than %d days.' % $._config.prowImageStaleByDays.daysStale,
```

Comment:
nit: Could also mention the `hoursToTrigger` in the description. (Otherwise it might seem like this alert is firing a day late.)

Reply:
Done